Task 1/2019201422李林豇/my_double.c

```diff
@@ -0,0 +1,704 @@
+#include<stdio.h>
+#include<math.h>
+#include<string.h>
+#include <stdint.h>  // uint64_t
+#include <stdbool.h> // bool
+
+#define lovelive  unsigned long long

```

Comment:
1. Don't declare type names with `#define`, use `typedef` instead.
2. Don't use a meaningless name for your type name.